lib/http/service.js

```diff
@@ -11,6 +11,16 @@
 // defined elsewhere into an actual Express service. The only thing it needs in
 // order to do this is a valid dependency injection context container.
 
+// REVIEW: per CONTRIBUTING.md, perhaps this should be rewritten?  Specifically:
```

Comment:
"sparingly" is still true and `cookie-parser` is stock parser of expressjs, right?